main_routers/system_router.py

```
@@ -512,6 +630,154 @@ async def ack_pending_notices(request: Request):
+
+
+@router.post("/tutorial-prompt/heartbeat")
+async def post_tutorial_prompt_heartbeat(request: Request):
+    """记录主页空闲与互动状态，并判断是否需要提示新手引导。"""
+    validation_error = _validate_local_mutation_request(request)
+    if validation_error is not None:
+        return validation_error
+
+    try:
+        payload = await request.json()
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)